main/middleware/apisix_user.py

```diff
@@ -99,6 +100,22 @@ def get_user_from_apisix_headers(request, decoded_headers, original_header):
             "get_user_from_apisix_headers: User %s not found, created new",
             global_id,
         )
+        # Send user creation event to PostHog
+        posthog = Posthog(
+            settings.POSTHOG_PROJECT_API_KEY, host=settings.POSTHOG_API_HOST
+        )
+        posthog.capture(
+            user.id,
+            event="user_account_created",
```

Comment:
Maybe we want to move this into a constants file? I don't think we do any other custom event capturing in Python yet, but if we're getting started on that it might be a good idea to get off on the right foot with organizing the keys for the different events in one place.

Reply:
Yes, I think its a good idea. I added PostHogEvents to define event constants. Can you take another look? Thanks